canvas_modules/common-canvas/src/toolbar/toolbar-button-item.jsx

```diff
@@ -18,9 +18,6 @@ import React from "react";
 import PropTypes from "prop-types";
 
 import Tooltip from "../tooltip/tooltip.jsx";
-import ArrangeHorizontally from "./../../assets/images/arrange_horizontally.svg";
```

Comment:
Can these be deleted from the file system?

Reply:
I have deleted these from the file system.